swap/src/database.rs

```diff
@@ -126,9 +127,12 @@ pub async fn open_db(
                 sqlite.insert_peer_id(swap_id, peer_id).await?;
             }
 
+            tracing::info!("Sucessfully migrated data to sqlite! Using sqlite.");
+
             Ok(Arc::new(sqlite))
         }
         (_, _, false) => {
+            tracing::debug!("Sled database not found. Using sqlite.");
             tokio::fs::File::create(&sqlite_path).await?;
```

Comment:
Wouldn't this fail if the sqlite DB already exists? I think we may need another pattern that checks for `sqlite` true and false and either doesn't or does create the file.